src/librustc/middle/region.rs

```diff
@@ -161,8 +161,7 @@ pub struct BlockRemainder {
 
 newtype_index!(FirstStatementIndex
     {
-        pub idx
-        MAX = SCOPE_DATA_REMAINDER_MAX
+        MAX = 0xFFFF_FFFB // the remainder of the scope data
```

Comment:
Can we do something less magic-value like than this? `0xFFFF_FFFB` is two steps away from the values in `SCOPE_DATA_*`. Changing the definition of `SCOPE_DATA_*` to be `0xFFFF_FFFE`, ... `0xFFFF_FFFD` would help somewhat, if it is not possible to use arithmetic in the attribute.

Reply:
`SCOPE_DATA_DESTRUCTION` is `0xFFFF_FFFC`, so this is exactly next to it.

Attributes currently only allow literals, no expressions. Though referring to constants might work via queries, there's currently no support for that.